.gitignore

```diff
@@ -25,7 +25,6 @@ packages/angular/projects/cds-angular/*
 # compiled output
 /dist
 /packages/**/dist
```

Comment:
Opened a todo issue to remind myself to move these over https://github.com/vmware-clarity/starters/issues/5